Containerfile

```diff
@@ -0,0 +1,36 @@
+ARG PYTHON_VERSION="3-slim"
```

Comment:
TIL about "Containerfile".  That said, my local runtime (docker mac) does not seem to find this file name as a default.  Can we keep with the ad-hoc standard name of "Dockerfile" for now?

Reply:
'Dockerfile' is slowly being faced out, because its not inclusive :) But i'll fix it.

btw, I can't reply to your previous comment about the guidelines, so I'll reply here.

If you look at https://github.com/docker-library/official-images#consistency
```
All official images should provide a consistent interface. A beginning user should be able to docker run official-image bash (or sh) without needing to learn about --entrypoint.
```
e.g. if you do
` docker run -it 281fd1590d41 /bin/sh`
it shouldn't work, which the guidelines say, the user is expected to do.

Reply:
> If you look at https://github.com/docker-library/official-images#consistency

Gotcha -- It seems as though we're in limbo between conditions #1 and #2
> 1. If the startup process does not need arguments, just use CMD
> 2. If there is initialization that needs to be done on start, like creating the initial database, use an ENTRYPOINT along with CMD

Amending my previous suggestion: ~ENTRYPOINT~ CMD ["semonitor.sh"] may be more appropriate.

Ironically, this doc on consistency is inconsistent itself: how would one support both of these scenarios without a wrapper script and thus how could suggestion #1 ever be complete?
> docker run official-image bash
> docker run official-image --arg1 --arg2

The several "good" examples references in the next section all use `CMD ["binary']` without a wrapper script.
https://github.com/docker-library/php/blob/b4aeb948e2e240c732d78890ff03285b16e8edda/5.6/Dockerfile#L65
https://github.com/docker-library/python/blob/3e5826ad0c6e29f07f6dc7ff8f30b4c54385d1bb/3.4/Dockerfile#L45

Do you think our situation is significantly different than these examples?

Reply:
I think you can't :) what's worse, dumb-init/tini even states you should avoid having single process containers (where you run daemons/servers).

putting `semonitor.sh` in command would basically do the same thing as the wrapper, but you still need the wrapper for the entrypoint :) so then you just are duplicating stuff.

But I agree, the docs aren't perfect, some texts are almost 10 years old without update.

I think most of those scripts however, are all short-lived processes. e.g. `jq` to give a nice example. We have a 'server' that runs forever. That's the major difference I suppose ...